locator/load_sketch.hh

```diff
@@ -137,6 +137,17 @@ public:
         std::make_heap(n._shards.begin(), n._shards.end(), shard_load_cmp());
     }
 
+    void pick(host_id node, shard_id shard) {
+        auto& n = _nodes.at(node);
+        for (auto& shard_load : n._shards) {
+            if (shard_load.id == shard) {
+                ++shard_load.load;
+                break;
+            }
```

Comment:
Isn't this wasteful? We have nodes with 200+ shards now.

Reply:
It is. I mirrored existing unload() which was added earlier.

Reply:
They are not use don the happy path, only when picked candidate is skipped due to constraints. So this is somewhat amortized.

Reply:
Fixed in V2